src/components/WebexMeeting/WebexMeeting.js

```diff
@@ -0,0 +1,46 @@
+import React, {Fragment} from 'react';
+import PropTypes from 'prop-types';
+import {Spinner} from '@momentum-ui/react';
+
+import {useMeeting} from '../hooks';
+
+import {WebexInMeeting, WebexInterstitialMeeting, WebexMeetingControl, WebexMeetingControls} from '..';
+
+import './WebexMeeting.css';
+
+/**
+ * Webex Meeting component displays all meeting contents.
+ *
+ * @param {object} props
+ * @returns {object} JSX of the component
+ */
+export default function WebexMeeting({meetingDestination, setControls}) {
+  const {ID, remoteVideo} = useMeeting(null, meetingDestination);
+
+  const isActive = remoteVideo !== null;
+
+  const controls = setControls(isActive).map((key) => <WebexMeetingControl type={key} key={key} />);
+  const meeting = ID ? (
+    <Fragment>
+      {isActive ? <WebexInMeeting meetingID={ID} /> : <WebexInterstitialMeeting meetingID={ID} />}
+      <div className="controls">
+        <WebexMeetingControls meetingID={ID}>{controls}</WebexMeetingControls>
+      </div>
+    </Fragment>
+  ) : (
+    <Spinner />
+  );
+
+  return <div className="meeting">{meeting}</div>;
```

Comment:
Can you move the `meeting` variable logic here? Not sure why it's separated out. It would also make the `Fragment` unnecessary

Reply:
we still need the `Fragment`. but I did move it in.